README.md

```diff
@@ -150,7 +150,7 @@ When you’ve tested the plugin and confirmed it’s working, notify your team s
    /github subscriptions add mattermost/mattermost-server --features issues,pulls,issue_comments,label:"Help Wanted"
    ```
   - The following flags are supported:
-     - `--features`: comma-delimited list of one or more of: issues, pulls, pulls_merged, pulls_created, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, label:"labelname". Defaults to pulls,issues,creates,deletes.
+     - `--features`: comma-delimited list of one or more of: issues, pulls, pulls_merged, pulls_created, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, workflows, label:"labelname". Defaults to pulls,issues,creates,deletes.
```

Comment:
I wonder if we should rename the flag to `workflow-failures` to make it clear that only these get notifications. Or add an option to `workflows` that specifies if only failed or also successful runs should trigger a notification.

Reply:
I'm thinking we should have separate `workflow-success` and `workflow-failure` subscription types

Reply:
@mickmister Added the `workflow_success`. Please have a look

Reply:
@hanzei are you still requesting changes here?

Reply:
Where did the `README.md` updates go?